### DISCUSSION TITLE
support

### DISCUSSION THREAD

**Discussion by Apersant1** [BepInEx/BepInEx#335] [Help and Q&A]
Will there be support for Windows 11?

**Comment by ghorsington** [comment]
BepInEx already works fine on Windows 11 and requires no additional setup steps:

![image](https://user-images.githubusercontent.com/13762037/142237504-45e8a131-47b1-4f3a-9196-2a66469bf9d2.png)

If you have any issues regarding BepInEx and Windows 11, please create a new issue ticket using Bug or Error report template. The templates include all the information you should provide for us to be able to debug the issue.
